pages/linux/dos2unix.md

```diff
@@ -0,0 +1,20 @@
+# dos2unix
+
+> DOS/Mac to UNIX and vice versa text file format converter.
+> Also check unix2dos
+
+- Convert a file in the DOS format to the UNIX format:
+
+`dos2unix {{path/to/file}}`
+
+- Convert and replace a file while keeping original date stamp:
+
+`dos2unix -k {{filename}}`
```

Comment:
Please use a consistent token style. Above, you have used path/to/file, here filename. Let us just use path/to/file everywhere. For cases of input and output file, have path/to/input_file and path/to/output_file